core/src/scss/components/card/_card--minimal.scss

```diff
@@ -0,0 +1,20 @@
+.su-card--minimal {
+  @include box-shadow($depth: 'flat', $stroke: 'none');
+  background-color: inherit;
+
+  .su-card__contents {
+    // override modular spacing at all breakpoints
+    @include grid-media-only(all) {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  &.su-card--horizontal {
+    // override modular spacing at all breakpoints
+    @include grid-media-only(all) {
+      padding-left: 0;
+      padding-right: 0;
+    }
```

Comment:
I think we could just do `padding: 0` for all sides for the horizontal minimal variant.

Reply:
done